bundle/DependencyInjection/Configuration/Parser/UserRegistration.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace EzSystems\RepositoryFormsBundle\DependencyInjection\Configuration\Parser;
+
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\AbstractParser;
+use eZ\Bundle\EzPublishCoreBundle\DependencyInjection\Configuration\SiteAccessAware\ContextualizerInterface;
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class UserRegistration extends AbstractParser
+{
+    /**
+     * Adds semantic configuration definition.
+     *
+     * @param \Symfony\Component\Config\Definition\Builder\NodeBuilder $nodeBuilder Node just under ezpublish.system.<siteaccess>
+     */
+    public function addSemanticConfig(NodeBuilder $nodeBuilder)
+    {
+        $nodeBuilder
+            ->arrayNode('users')
+                ->info('Users configuration')
+                ->children()
+                    ->scalarNode('registration_group_id')
+                        ->info('Content id of the user group where users who register are created.')
+                        ->defaultValue(11)
```

Comment:
> This will be doc-ed somewhere, I presume?

Well, it is already documented when running `debug:configuration ezpublish` ;)
But yes, it will require its own entry in the doc, indeed.

> Also, will you add the setting to platform repo config files, where people will discover it on their own?

I didn't really intend to do that, but we could, no strong opinion.
Maybe @andrerom has an opinion about that ?